diff --color -urN sx1302_hal-2.1.0.orig/libloragw/tst/test_loragw_cal_sx125x.c sx1302_hal-2.1.0/libloragw/tst/test_loragw_cal_sx125x.c
--- sx1302_hal-2.1.0.orig/libloragw/tst/test_loragw_cal_sx125x.c	2022-02-11 19:15:34.167321202 +0200
+++ sx1302_hal-2.1.0/libloragw/tst/test_loragw_cal_sx125x.c	2022-02-11 19:23:31.609712560 +0200
@@ -593,7 +593,7 @@
 
     if (com_type == LGW_COM_SPI) {
         /* Board reset */
-        if (system("./reset_lgw.sh start") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh start") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
@@ -648,7 +648,7 @@
 
     if (com_type == LGW_COM_SPI) {
         /* Board reset */
-        if (system("./reset_lgw.sh stop") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh stop") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
diff --color -urN sx1302_hal-2.1.0.orig/libloragw/tst/test_loragw_com.c sx1302_hal-2.1.0/libloragw/tst/test_loragw_com.c
--- sx1302_hal-2.1.0.orig/libloragw/tst/test_loragw_com.c	2022-02-11 19:15:34.167321202 +0200
+++ sx1302_hal-2.1.0/libloragw/tst/test_loragw_com.c	2022-02-11 19:23:31.601712453 +0200
@@ -125,7 +125,7 @@
 
     /* Board reset */
     if (com_type == LGW_COM_SPI) {
-        if (system("./reset_lgw.sh start") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh start") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
@@ -373,7 +373,7 @@
 
     if (com_type == LGW_COM_SPI) {
         /* Board reset */
-        if (system("./reset_lgw.sh stop") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh stop") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
diff --color -urN sx1302_hal-2.1.0.orig/libloragw/tst/test_loragw_com_sx1250.c sx1302_hal-2.1.0/libloragw/tst/test_loragw_com_sx1250.c
--- sx1302_hal-2.1.0.orig/libloragw/tst/test_loragw_com_sx1250.c	2022-02-11 19:15:34.167321202 +0200
+++ sx1302_hal-2.1.0/libloragw/tst/test_loragw_com_sx1250.c	2022-02-11 19:23:31.593712345 +0200
@@ -116,7 +116,7 @@
 
     /* Board reset */
     if (com_type == LGW_COM_SPI) {
-        if (system("./reset_lgw.sh start") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh start") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
@@ -210,7 +210,7 @@
 
     if (com_type == LGW_COM_SPI) {
         /* Board reset */
-        if (system("./reset_lgw.sh stop") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh stop") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
diff --color -urN sx1302_hal-2.1.0.orig/libloragw/tst/test_loragw_com_sx1261.c sx1302_hal-2.1.0/libloragw/tst/test_loragw_com_sx1261.c
--- sx1302_hal-2.1.0.orig/libloragw/tst/test_loragw_com_sx1261.c	2022-02-11 19:15:34.167321202 +0200
+++ sx1302_hal-2.1.0/libloragw/tst/test_loragw_com_sx1261.c	2022-02-11 19:23:31.605712508 +0200
@@ -116,7 +116,7 @@
 
     /* Board reset */
     if (com_type == LGW_COM_SPI) {
-        if (system("./reset_lgw.sh start") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh start") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
@@ -194,7 +194,7 @@
 
     if (com_type == LGW_COM_SPI) {
         /* Board reset */
-        if (system("./reset_lgw.sh stop") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh stop") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
diff --color -urN sx1302_hal-2.1.0.orig/libloragw/tst/test_loragw_counter.c sx1302_hal-2.1.0/libloragw/tst/test_loragw_counter.c
--- sx1302_hal-2.1.0.orig/libloragw/tst/test_loragw_counter.c	2022-02-11 19:15:34.167321202 +0200
+++ sx1302_hal-2.1.0/libloragw/tst/test_loragw_counter.c	2022-02-11 19:23:31.601712453 +0200
@@ -183,7 +183,7 @@
 
     if (com_type == LGW_COM_SPI) {
         /* Board reset */
-        if (system("./reset_lgw.sh start") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh start") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
@@ -265,7 +265,7 @@
 
     if (com_type == LGW_COM_SPI) {
         /* Board reset */
-        if (system("./reset_lgw.sh stop") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh stop") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
diff --color -urN sx1302_hal-2.1.0.orig/libloragw/tst/test_loragw_gps.c sx1302_hal-2.1.0/libloragw/tst/test_loragw_gps.c
--- sx1302_hal-2.1.0.orig/libloragw/tst/test_loragw_gps.c	2022-02-11 19:15:34.167321202 +0200
+++ sx1302_hal-2.1.0/libloragw/tst/test_loragw_gps.c	2022-02-11 19:23:31.609712560 +0200
@@ -261,7 +261,7 @@
 
     if (com_type == LGW_COM_SPI) {
         /* Board reset */
-        if (system("./reset_lgw.sh start") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh start") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
@@ -416,7 +416,7 @@
 
     if (com_type == LGW_COM_SPI) {
         /* Board reset */
-        if (system("./reset_lgw.sh stop") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh stop") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
diff --color -urN sx1302_hal-2.1.0.orig/libloragw/tst/test_loragw_hal_rx.c sx1302_hal-2.1.0/libloragw/tst/test_loragw_hal_rx.c
--- sx1302_hal-2.1.0.orig/libloragw/tst/test_loragw_hal_rx.c	2022-02-11 19:15:34.167321202 +0200
+++ sx1302_hal-2.1.0/libloragw/tst/test_loragw_hal_rx.c	2022-02-11 19:23:31.605712508 +0200
@@ -363,7 +363,7 @@
 
         if (com_type == LGW_COM_SPI) {
             /* Board reset */
-            if (system("./reset_lgw.sh start") != 0) {
+            if (system("/usr/sbin/reset_lgw.sh start") != 0) {
                 printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
                 exit(EXIT_FAILURE);
             }
@@ -423,7 +423,7 @@
 
         if (com_type == LGW_COM_SPI) {
             /* Board reset */
-            if (system("./reset_lgw.sh stop") != 0) {
+            if (system("/usr/sbin/reset_lgw.sh stop") != 0) {
                 printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
                 exit(EXIT_FAILURE);
             }
diff --color -urN sx1302_hal-2.1.0.orig/libloragw/tst/test_loragw_hal_tx.c sx1302_hal-2.1.0/libloragw/tst/test_loragw_hal_tx.c
--- sx1302_hal-2.1.0.orig/libloragw/tst/test_loragw_hal_tx.c	2022-02-11 19:15:34.167321202 +0200
+++ sx1302_hal-2.1.0/libloragw/tst/test_loragw_hal_tx.c	2022-02-11 19:23:31.605712508 +0200
@@ -480,7 +480,7 @@
     for (cnt_loop = 0; cnt_loop < nb_loop; cnt_loop++) {
         if (com_type == LGW_COM_SPI) {
         /* Board reset */
-            if (system("./reset_lgw.sh start") != 0) {
+            if (system("/usr/sbin/reset_lgw.sh start") != 0) {
                 printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
                 exit(EXIT_FAILURE);
             }
@@ -600,7 +600,7 @@
 
         if (com_type == LGW_COM_SPI) {
             /* Board reset */
-            if (system("./reset_lgw.sh stop") != 0) {
+            if (system("/usr/sbin/reset_lgw.sh stop") != 0) {
                 printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
                 exit(EXIT_FAILURE);
             }
diff --color -urN sx1302_hal-2.1.0.orig/libloragw/tst/test_loragw_reg.c sx1302_hal-2.1.0/libloragw/tst/test_loragw_reg.c
--- sx1302_hal-2.1.0.orig/libloragw/tst/test_loragw_reg.c	2022-02-11 19:15:34.167321202 +0200
+++ sx1302_hal-2.1.0/libloragw/tst/test_loragw_reg.c	2022-02-11 19:23:31.597712399 +0200
@@ -95,7 +95,7 @@
 
     if (com_type == LGW_COM_SPI) {
         /* Board reset */
-        if (system("./reset_lgw.sh start") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh start") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
@@ -190,7 +190,7 @@
 
     if (com_type == LGW_COM_SPI) {
         /* Board reset */
-        if (system("./reset_lgw.sh stop") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh stop") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
diff --color -urN sx1302_hal-2.1.0.orig/libloragw/tst/test_loragw_sx1261_rssi.c sx1302_hal-2.1.0/libloragw/tst/test_loragw_sx1261_rssi.c
--- sx1302_hal-2.1.0.orig/libloragw/tst/test_loragw_sx1261_rssi.c	2022-02-11 19:15:34.167321202 +0200
+++ sx1302_hal-2.1.0/libloragw/tst/test_loragw_sx1261_rssi.c	2022-02-11 19:23:31.565711969 +0200
@@ -201,7 +201,7 @@
 
     /* Board reset */
     if (com_type == LGW_COM_SPI) {
-        if (system("./reset_lgw.sh start") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh start") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
@@ -308,7 +308,7 @@
 
     if (com_type == LGW_COM_SPI) {
         /* Board reset */
-        if (system("./reset_lgw.sh stop") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh stop") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
diff --color -urN sx1302_hal-2.1.0.orig/packet_forwarder/src/lora_pkt_fwd.c sx1302_hal-2.1.0/packet_forwarder/src/lora_pkt_fwd.c
--- sx1302_hal-2.1.0.orig/packet_forwarder/src/lora_pkt_fwd.c	2022-02-11 19:15:34.171321257 +0200
+++ sx1302_hal-2.1.0/packet_forwarder/src/lora_pkt_fwd.c	2022-02-11 19:23:31.561711914 +0200
@@ -1641,7 +1641,7 @@
 
     if (com_type == LGW_COM_SPI) {
         /* Board reset */
-        if (system("./reset_lgw.sh start") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh start") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
@@ -1939,7 +1939,7 @@
 
     if (com_type == LGW_COM_SPI) {
         /* Board reset */
-        if (system("./reset_lgw.sh stop") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh stop") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
diff --color -urN sx1302_hal-2.1.0.orig/util_chip_id/src/chip_id.c sx1302_hal-2.1.0/util_chip_id/src/chip_id.c
--- sx1302_hal-2.1.0.orig/util_chip_id/src/chip_id.c	2022-02-11 19:15:34.171321257 +0200
+++ sx1302_hal-2.1.0/util_chip_id/src/chip_id.c	2022-02-11 19:23:31.685713583 +0200
@@ -150,7 +150,7 @@
 
     if (com_type == LGW_COM_SPI) {
         /* Board reset */
-        if (system("./reset_lgw.sh start") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh start") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
@@ -214,7 +214,7 @@
 
     if (com_type == LGW_COM_SPI) {
         /* Board reset */
-        if (system("./reset_lgw.sh stop") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh stop") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
diff --color -urN sx1302_hal-2.1.0.orig/util_spectral_scan/src/spectral_scan.c sx1302_hal-2.1.0/util_spectral_scan/src/spectral_scan.c
--- sx1302_hal-2.1.0.orig/util_spectral_scan/src/spectral_scan.c	2022-02-11 19:15:34.171321257 +0200
+++ sx1302_hal-2.1.0/util_spectral_scan/src/spectral_scan.c	2022-02-11 19:23:31.777714822 +0200
@@ -215,7 +215,7 @@
 
     if (com_type == LGW_COM_SPI) {
         /* Board reset */
-        if (system("./reset_lgw.sh start") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh start") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
@@ -355,7 +355,7 @@
 
     if (com_type == LGW_COM_SPI) {
         /* Board reset */
-        if (system("./reset_lgw.sh stop") != 0) {
+        if (system("/usr/sbin/reset_lgw.sh stop") != 0) {
             printf("ERROR: failed to reset SX1302, check your reset_lgw.sh script\n");
             exit(EXIT_FAILURE);
         }
